packages/grid/x-data-grid/src/hooks/features/virtualization/useGridVirtualScroller.tsx

```diff
@@ -453,6 +460,29 @@ export const useGridVirtualScroller = (props: UseGridVirtualScrollerProps) => {
           columns: visibleColumns,
         });
       }
+
+      // If the selected row is not within the current range of rows being displayed,
```

Comment:
The `rowThreshold` doesn't work as expected. You can see the focused cell before virtualized rows are rendered: https://deploy-preview-7357--material-ui-x.netlify.app/x/react-data-grid/virtualization/

https://user-images.githubusercontent.com/13808724/212136668-2c878c74-67a3-4681-a9d3-abcd64dad4f8.mov

Reply:
I have set the row width and opacity to zero. it seems to fixe the problem

Reply:
Yes, I cannot see the row with the focused cell, but now I see the blank space, which shouldn't happen either:

https://user-images.githubusercontent.com/13808724/219099182-6e8c1ea0-03ff-4662-be83-5eabfbbb3076.mov

Compared to https://deploy-preview-7357--material-ui-x.netlify.app/x/react-data-grid/virtualization/ where it doesn't happen:

https://user-images.githubusercontent.com/13808724/219099683-eaea2a3b-e5bf-4b57-b045-36a9a776591d.mov

Because the focused row is rendered when it's out of the viewport, it shifts the rows below it and therefore breaks the rowBuffer behavior

Reply:
I forgot to set the row height to 0. its fixed now

Reply:
I think it's resolved now.